libs/martech/martech.js

```diff
@@ -1,7 +1,7 @@
-import { getConfig, loadIms, loadLink, loadScript } from '../utils/utils.js';
+import { getConfig, getMetadata, loadIms, loadLink, loadScript } from '../utils/utils.js';
 
 const ALLOY_SEND_EVENT = 'alloy_sendEvent';
-const TARGET_TIMEOUT_MS = 2000;
+const TARGET_TIMEOUT_MS = 3500;
```

Comment:
I share @mokimo's concerns regarding performance impact of increasing this value. Hopefully we can bring it back down with self-hosted IMS and martech scripts

Reply:
@overmyheadandbody @mokimo 
Yeah, we are doing a few things at once:
1. increasing the timeout because we strongly suspect 2 seconds is too short.
2. adding an analytic event to get quickly a sense of how many more users we get for every .25 seconds longer we wait. 3. Also trying to ascertain our current fail rate.  The hope is that this is a temporary measure (couple of months)
4. creating a grafana dashboard to track LHS on the same URL's with the target-timeout parameter dictating different wait times.
5. add a permanent lana log metric so we can monitor spikes in Target response times

But we're trying to get data to find the "goldilocks" timeout.  Wait too little and too many people don't get the Target activity.  Too long and we're waiting for a few fringe users.